minitest/models.py

```
@@ -0,0 +1,50 @@
+    """회사 모델"""
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    # 참조된 Name 객체가 삭제되면 이 회사도 삭제됨 
+    user_id = models.ForeignKey(Name, on_delete=models.CASCADE, related_name='companies')
+    company_name = models.CharField(max_length=100)
+    company_address = models.CharField(max_length=200, blank=True, null=True)
+    total_staff = models.IntegerField(default=0, validators=[MinLengthValidator(0), MaxLengthValidator(10000)])
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def save(self, *args, **kwargs):
+            self.clean()
```

Pylint (3.3.7)

[warning] Bad indentation. Found 12 spaces, expected 8

(W0311)